src/main.js

```diff
@@ -3,7 +3,6 @@
 // ---------------------------------------------------
 
 import express from "express";
-import fs from "fs";
```

Comment:
Ok this. But bear in mind that the private key we are using in the .env will be possibly moved to a secure storage, such as AWS Secrets Manager. This is the way to go to secure `.pem` files. Take a look at this [link](https://medium.com/@munteanu210/what-is-a-pem-file-navigating-its-creation-use-and-conversion-in-ssl-a7ac689867ae) to understand what is a pem file.

Reply:
I removed this import only because we weren't using it in the file. I'm fine with adding it back.